externs/shaka/player.js

```diff
@@ -615,12 +615,16 @@ shaka.extern.DashManifestConfiguration;
 
 /**
  * @typedef {{
- *   ignoreTextStreamFailures: boolean
+ *   ignoreTextStreamFailures: boolean,
+ *   forceFullSegmentRequest: boolean
  * }}
  *
  * @property {boolean} ignoreTextStreamFailures
  *   If <code>true</code>, ignore any errors in a text stream and filter out
  *   those streams.
+ * @property {boolean} forceFullSegmentRequest
+ *   If <code>true</code>, force HlsParser to use a full segment request in
```

Comment:
Similarly, this should be reworded to make it clear that it specifically makes it load full segments to determine start time.

Reply:
done